index.bs

```diff
@@ -730,6 +822,15 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} of the {{VRDisplay}} this {{Gamepad}} is associated with. A {{Gamepad}} is considered to be associated with a {{VRDisplay}} if it reports a pose that is in the same space as the {{VRDisplay}} pose. If the {{Gamepad}} is not associated with a {{VRDisplay}} should return 0.
 
+<dfn attribute for="Gamepad">hasPosition</dfn>
+The {{Gamepad}}.{{Gamepad/hasPosition}} attribute MUST return whether the {{Gamepad}}'s position is capable of being tracked.
+
+<dfn attribute for="Gamepad">hasOrientation</dfn>
+The {{Gamepad}}.{{Gamepad/hasOrientation}} attribute MUST return whether the {{Gamepad}}'s orientation is capable of being tracked.
+
+<dfn method for="Gamepad">getPose()</dfn>
+Retrieves a {{VRPose}} in the supplied {{VRCoordinateSystem}} for the current {{VRFrameData}}.  This function will return false if the Gamepad's pose cannot be expressed in the supplied {{VRCoordinateSystem}} and will return true otherwise.
```

Comment:
Gamepad's can gain value from being tracked at a faster refresh rate or separately from the HMD in some cases allowing for better embodied presence. Is it correct then to state that the Gamepad's pose must be related to the current VRFrameData?

Maybe this is how it has to be for Web VR to make sense though, some normalization process has to occur otherwise having Gamepad's tracked in completely different ways from the HMD and maybe other tracked objects would be hard to reason about.